### ISSUE TITLE
some bioconductor package can not install automaticly

### ISSUE THREAD

**Issue by Solipugids** [wind22zhu/BioMedR#1]
When I installed BioMedR through CRAN, got the error message for some dependencies like "fmcsR", "ChemmineR", "GoSemSim". 

> ERROR: dependency '"fmcsR", "ChemmineR", "GoSemSim"' is not available for package 'BioMedR' 

 It looks like it was brought by different repo between CRAN and Bioconductor.  I have to install the dependencies manually.  

Also, I have a question that what is the differences of BioMedR with protr. "protr" is also your contributions and there are overlapping functions in these two packages. 

Thank you.

**Comment by wind22zhu** [comment]
@Solipugids Yes, we have ever released the Bioconductor version of BioMedR. Now we only release it on CRAN.
1) Some dependencies should be installed manually when you get the tips.
2) We updated the installation notes.
3) protr only provides functions to process protein sequences. BioMedR can calculate various molecular descriptors for chemicals, proteins, DNAs/RNAs and their interactions. Also, this package realized some similarity searching methods and powerful clustering methods as well as several useful auxiliary tools.
